File: HaykAmirjanyan/week1 continue/caesar.c
#include <stdio.h>
#include <cs50.h>
#include <string.h>

int main(void) {
    
    int key = get_int("write a key > "), i, c;
    string word = get_string("write a word > ");
    
    key = key % 26;
    
    for(i = 0; i < strlen(word); i++) {
       
        c = (int)word[i] + key;
        
       if(( (int)word[i] <= 90 && c > 90 ) || ( (int)word[i] >= 97 && c > 122) ) {
            
               c = c - 26;
       }
       
        word[i] = (char)c;
    }
    
    printf("%s\n", word); 
}

File: HaykAmirjanyan/pset6/Bleep/bleep/caesar.py
from cs50 import get_string
from sys import argv
import sys
if len(sys.argv) > 1:
    n = int(argv[1])
else:
    print("no key")
    exit(1)
if n < 0:
    print("Write positive number please: ")
    exit(1)

sentance = get_string("Write a sentance: ")
if n > 26:
    n = n % 26
outsentance = ""
for i in range(len(sentance)):
    m = ord(sentance[i])
    if sentance[i].isalpha():
        m += n
        if (sentance[i].isupper() and m > 90) or (sentance[i].islower() and m > 122):
            m -= 26
    outsentance += chr(m)
print("ciphertext:" + outsentance)